components/TeamLeaderboard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Image from 'next/image';
+
+export default function TeamLeaderboard() {
+
+    const data = [
+        {
+            rank: 1,
+            talentScore: 1016.79,
+            name: "Alabama",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/333.png",
+            conference: "SEC"
+        },
+        {
+            rank: 2,
+            talentScore: 989.76,
+            name: "Georgia",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/61.png",
+            conference: "SEC"
+        },
+        {
+            rank: 3,
+            talentScore: 983.14,
+            name: "Ohio State",
+            image: "https://a.espncdn.com/i/teamlogos/ncaa/500/194.png",
+            conference: "Big Ten"
+        }
+    ]
+
+    return (
+        <div>
+            <>            
+                <div className="grid grid-cols-4 ml-28 mr-28 rounded-md text-3xl mt-10 font-bold text-gray-200 justify-items-center">  
+                    <h1>Rank</h1>                
+                    <h1>Team</h1>                
+                    <h1>Talent Score</h1>                
+                    <h1>Conference</h1>            
+                </div>           
+                <ul>
+                    {data.map((team) => (
```

Comment:
Make each one of these a link to the team itself can use just an onclick event in that first div and ask cole for some real data then I think this is good to go